### PR TITLE
Added cluster overview grafana dashboard

### PR DESCRIPTION
**Pull request by afritzler**
* Added new cluster overview Grafana dashboard based on [1]
* Adjusted Promehteus scraping Rules according to [1]

[1] https://github.com/instrumentisto/grafana-dashboard-kubernetes-prometheus

### CODE REVIEW

File: modules/seed/addons/monitoring/templates/prometheus-config.yaml [hunk 1]
```diff
@@ -203,4 +203,38 @@ scrape_configs:
     replacement: $1:$2
     target_label: __address__
   - action: labelmap
-    regex: __meta_kubernetes_pod_label_(.+)
\ No newline at end of file
+    regex: __meta_kubernetes_pod_label_(.+)
+
+# Needed for Grafana Dashboard
+# from https://github.com/instrumentisto/grafana-dashboard-kubernetes-prometheus
+- job_name: kubernetes-nodes-cadvisor
```
Comment (dkistner) [inline modules/seed/addons/monitoring/templates/prometheus-config.yaml]: This scrape job does not work for me. Prometheus get a 404 when trying to scrape the target of this job.
But it seems to me that we have already a job for the cadvisor (``job=kubernetes-cadvisor``), which is working, see below.

```
https://kubernetes.default.svc:443/api/v1/nodes/1/proxy/metrics/cadvisor [DOWN, job=kubernetes-nodes-cadvisor]
https://kubernetes.default.svc:443/api/v1/nodes/ip-10-250-149-142.eu-central-1.compute.internal/proxy/metrics/cadvisor [UP, job=kubernetes-cadvisor]
```
Could we use this job instead? I think then we have to adapt the queries in the dashboard?

Comment (afritzler) [reply modules/seed/addons/monitoring/templates/prometheus-config.yaml]: Well I could remove that one. I added the scrape config according to the documentation of the k8s dashboard.

Comment (afritzler) [reply modules/seed/addons/monitoring/templates/prometheus-config.yaml]: Ok, the scraping is working without this job.

**Comment by dkistner** [comment]
LGTM

**Pull request merged**